Dockerfile

```diff
@@ -34,9 +34,9 @@ RUN apk add --no-cache python3==${PYTHON3_VERSION} && \
     curl -Lo /usr/local/bin/yq "https://github.com/mikefarah/yq/releases/download/2.4.0/yq_linux_amd64" && \
     chmod +x /usr/local/bin/yq && \
     curl -sL https://github.com/digitalocean/doctl/releases/download/v${DOCTL_VERSION}/doctl-${DOCTL_VERSION}-linux-amd64.tar.gz | tar -xzv && \
-    apk add libc6-compat && \
+    apk add libc6-compat sudo util-linux&& \
```

Comment:
unfortunately, git-extras-setup does require 'sudo' which is why i am installing it here.

Reply:
i dont think it makes sense to have this in a digital ocean docker image. This is for git operation. I don't think we need this do do ci stuff to deploy to digital ocean